spring-restdocs-asciidoctor/src/main/resources/extensions/srd-include-block-macro.rb

```diff
@@ -0,0 +1,103 @@
+require 'asciidoctor/extensions'
+require 'stringio'
+
+# Spring REST Docs block macro to import multiple snippet of an operation at once
+#
+# Usage
+#
+#   srd::operation-name[snippets='snippet-name1,snippet-name2', level=<indentation level>]
+#
+class RestDocsBlockMacro < Asciidoctor::Extensions::BlockMacroProcessor
```

Comment:
I think this should be called `OperationBlockMacro` and the file itself should be called `operation_block_macro.rb`